packages/babel-preset/esm.js

```diff
@@ -0,0 +1,49 @@
+const envPreset = require('@babel/preset-env');
+const modulesPreset = require('@babel/preset-modules');
+const reactPreset = require('@babel/preset-react');
+const plugins = require('./plugins');
+
+function preset(api, options = {}) {
+  const {
+    runtime,
+    useBuiltIns,
+    debug,
+    loose = true,
+    modules = api.env() === 'test',
+    development = api.env() === 'test',
+  } = options;
+
+  const presets = [
+    [modulesPreset, { loose }],
+
+    useBuiltIns && [
+      envPreset,
+      {
+        debug,
+        loose,
+        targets: { esmodules: true },
+        exclude: [/transform/],
```

Comment:
what does this do?

Reply:
exludes all the transforms that the preset selects for but leaves the proposals and the polyfills. preset-modules covers the transforms for what is excluded

Reply:
okay, could you add a comment noting that this preset-env then is for just the proposals? because it's fairly different from e.g. what the preset-modules docs show

it's not obvious just looking at this

Reply:
should still have a comment here

Reply:
Sorry thought I added one